definitions/npm/deepmerge_v2.1.x/flow_v0.57.3-/deepmerge_v2.1.x.js

```diff
@@ -0,0 +1,12 @@
+declare module 'deepmerge' {
+  declare type Options = {
+    clone?: boolean,
+    arrayMerge?: (destination: any[], source: any[], options?: Options) => Array<any>
+  }
+
+  declare module.exports: {
+    <A, B>(a: A, b: B, options?: Options): A & B,
+    all<T>(objects: Array<$Shape<T>>, options?: Options): T
```

Comment:
This def requires the objects to be all the same shape, which does not reflect the docs.